.github/workflows/publish-packages.yaml

```diff
@@ -0,0 +1,64 @@
+name: Publish
+on:
+  # NOTE: avoid using the manual execution unless is the only way to fix some issue,
+  # rather retry failed jobs in case of flakes. The manual execution can potentially
+  # cause the tags to point to a different commit that the one used to publish
+  # the packages
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["Check and Test"]
```

Comment:
this is not going to work here, should we maybe run just unit tests here? ignore tests .. or, any other ideas? cc: @gribnoysup

Reply:
I would really like to not have multiple places where we run tests again as we had before. I would even prefer that we do this all CI stuff in evergreen, but I get that it's really not easy to do for things like that. If we are opening a PR to bump versions and evergreen is green there, it feels alright to me to just publish when this version bump PR is merged to main

Reply:
We probably should run tests there then, I guess we can't use the bot here though, otherwise that won't trigger

Reply:
If we want to move tests to GH again then we need to somehow figure out how to orchestrate releases between CI systems, because we can't move application publishing out of Evergreen

Reply:
I mean in the PR that gets merged, should be enough to use devtoolsbot, and then the PR that is opened is tested normally with EVG

Reply:
Okay, I think I didn't get you right the first time and maybe still not understanding correctly, I'm sorry 😆 So from my perspective I think it would be okay to ignore tests here and just publish because presumably we already ran tests enough times before this landed in main, but if you think it's a bit unsafe maybe it would not be that bad if we run only unit tests in this task, I would really not try to move all "check and test" tasks back to github after we moved them to evergreen. What I think I'm missing is how the devtoolsbot is part of this 🤔

Reply:
> I would really not try to move all "check and test" tasks back

Oh, I wasn't seriously proposing to move tests back on Github Actions, that "Check and Test" is a copy-paste from the other repo, that's why I was asking for ideas.

>  I'm missing is how the devtoolsbot is part of this

One thing that came to my mind was that we could skip the test on "publish" completely, but we could then run all the regular tests on the "bump" PR as we would normally do for the other PRs. Currently, though, tests are skipped in the "bump" PR cause is opened by the GitHub bot, which is special and doesn't trigger any automation. That's why I mentioned using the other bot here.

For completeness: there are marketplace actions that would let us wait for all the checks to pass, but buh, no idea how well it works without a PR, I would need to research a bit more.

I'd pick one of:
- no tests at all
- test the bump PR (only cause is actually possible that it was created from a broken main, before tests passed there)

Reply:
Ahhh, got it now, thank you! I totally forgot that non-org accounts don't trigger the CI!

I feel like testing the bump PR would be a very good approach, but if it's an impossible stretch to make it work I think I would feel alright if we weren't testing these bump PRs

Reply:
@gribnoysup I'm actually reconsidering this a bit .. since we are not forced to publish packages from Github Actions and considering what we wanted to investigate to improve the integration with DE / Clould, it may make more sense to publish packages from EVG at the end of a pipeline from main.

We may be able to decide wether to publish a prerelease / sha based tag, by looking at the commit message:

ie. 'ci bump packages': --> publish the actual tags
other commit messages --> publish with "sha"

Reply:
I changed my mind, let's start with this for now as is almost ready, we can move things to EVG later